.github/workflows/build-notebooks-TEMPLATE.yaml

```diff
@@ -101,22 +107,45 @@ jobs:
 
       - name: Configure Podman
         run: |
-          set -x
-          mkdir -p $HOME/.config/containers/
-          cp ci/cached-builds/containers.conf $HOME/.config/containers/containers.conf
-          cp ci/cached-builds/storage.conf $HOME/.config/containers/storage.conf
-
-          # should at least reset storage when touching storage.conf
-          podman system reset --force
+          set -Eeuxo pipefail
+
+          # podman running as service ignores the TMPDIR env var here, let's give it a bind-mount to /var/tmp
+          mkdir -p $TMPDIR
+          sudo mount --bind -o rw,noexec,nosuid,nodev,bind $TMPDIR /var/tmp
+
+          # podman from brew has its own /etc (was giving me Failed to obtain podman configuration: runroot must be set)
+          # the (default) config location is also where cri-o gets its storage defaults (that can be overriden in crio.conf)
+          sudo cp ci/cached-builds/containers.conf /etc/containers.conf
+          sudo cp ci/cached-builds/containers.conf /home/linuxbrew/.linuxbrew/opt/podman/etc/containers.conf
+          sudo cp ci/cached-builds/storage.conf /etc/containers/storage.conf
+          sudo cp ci/cached-builds/storage.conf /home/linuxbrew/.linuxbrew/opt/podman/etc/containers/storage.conf
+          sudo cp ci/cached-builds/registries.conf /etc/containers/registries.conf
+          sudo cp ci/cached-builds/registries.conf /home/linuxbrew/.linuxbrew/opt/podman/etc/containers/registries.conf
```

Comment:
these are just the default config files, but I need to have some nondefault values in them, mostly the path to containers/storage; it's pretty much the same as it was before, but i'm putting them into the systemwide location, not just into user directory, because now we run rootful podman

also, I am putting it into the systemwide locations for crio as well as podman, and because podman is installed from homebrew, it lives under /home/linuxbrew/.linuxbrew/opt/podman/etc; weird but so is life